git_trunk/tests/common.py

```diff
@@ -44,6 +44,23 @@ def _build_repo(self, git, dir_local, dir_remote):
         # Get back to master.
         git.checkout('master')
 
+    def _create_dummy_submodule(self, dir_name):
+        """
```

Comment:
Looks like you are not using git-hooks:)
This is incorrect docstring format.

It violates pydocstyle: `D200 One-line docstring should fit on one line with quotes`